features/support/generic_helpers.rb

```diff
@@ -100,10 +100,18 @@ def summary_page
   @summary_page ||= SummaryPage.new
 end
 
+def summary_page_error
+  @summary_page ||= SummaryPageError.new
+end
```

Comment:
This should be @summary_page_error.  This is related how ruby's attribute accessor works. 
But based on quick look at the SummaryPageError page this page object doesn't have to exist at all.